app/bundles/ProjectBundle/Tests/Functional/Controller/ProjectEntityLookupLimitTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\ProjectBundle\Tests\Functional\Controller;
+
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\EmailBundle\Entity\Email;
+use Mautic\EmailBundle\Model\EmailModel;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProjectEntityLookupLimitTest extends MauticMysqlTestCase
+{
+    private const LOOKUP_CHOICE_LIST_URL = '/s/ajax?action=project:getLookupChoiceList';
+
+    /**
+     * Create 2000 test emails.
+     */
+    private function createTestEmails(): void
```

Comment:
The order of methods should be in a class:
1. public
2. protected
3. public

Please move the private method to the bottom.